src/app/components/contextmenu/contextmenu.ts

```diff
@@ -607,7 +607,7 @@ export class ContextMenu implements OnInit, AfterContentInit, OnDestroy {
             if (!this.documentClickListener) {
                 const documentTarget: any = this.el ? this.el.nativeElement.ownerDocument : 'document';
 
-                this.documentClickListener = this.renderer.listen(documentTarget, 'click', (event) => {
+                this.documentClickListener = this.renderer.listen(documentTarget, 'mousedown', (event) => {
                     if (this.containerViewChild.nativeElement.offsetParent && this.isOutsideClicked(event) && !event.ctrlKey && event.button !== 2 && this.triggerEvent !== 'click') {
```

Comment:
Line 610 sets listener from **click** to **mousedown**

Line 611 still references **click**.  Is this correct?